### ISSUE TITLE
mGBA core broken in main RA app

### ISSUE THREAD

**Issue by roflcopter777** [libretro/RetroArch#4522]
**Description**

Loading the mGBA core via the centralized RA app loads perfectly fine, but when you load a ROM via
said core, the Wii U locks up with a DSI error. mGBA builds used outside of the main app function perfectly fine, as to why the main RA app causes it to crash is a mystery. 

**Expected behavior**

The mGBA core should  load a ROM fine irrespective of it being loaded either via its own app or the main RA app. 

**Actual behavior**

Core loads up fine as stated before, but loading a ROM causes immediate lockup of the console, forcing a hard reboot. 

**Steps to reproduce the bug**

1. Load RetroArch 1.4.0 (and any subsequent builds released after)
2. Select and load the mGBA core
3. Load up any ROM

**Bisect Results**

This issue has been occurring since the 1/25/17 builds, I currently have a backup of builds
compiled on 1/17/17, and other users report using the standalone mGBA RA core/app working
perfectly fine, it only locks up when you load the core via 1.4.0.  This indicates something
is indeed broken for whatever reason.

**Version/Commit**

- RetroArch: Version 1.4.0
https://buildbot.libretro.com/nightly/nintendo/wiiu/2017-01-25_RetroArch_rpx.7z

**Comment by roflcopter777** [comment]
Apparently a few other cores are broken in the latest nightly builds

**Comment by aliaspider** [comment]
it turned out to be a core issue afterall. the mgba build broke after the toolchain update. 
998dc43314e6572c8e651d0f46afbefcf7d6aa47
can you check if this fixes the issue for you when the new builds are up ?
also, you said other cores were affected, do you have a list ?

**Comment by roflcopter777** [comment]
Intriguing, probably good that I reported it, and admittedly I was hesitant because you guys are so busy squashing bugs and so on.  Unfortunately, I don't have a comprehensive list per se, but other cores affected include Stella, VECX, and Virtual Jaguar. At least, that is what I've gathered from the thread on GBA Temp regarding the issue. I haven't had a chance to test out the builds today, due to being gone for work. I will test them ASAP and report back. Thank you.
